Objects/unionobject.c

```diff
@@ -440,6 +367,12 @@ union_getattro(PyObject *self, PyObject *name)
     return PyObject_GenericGetAttr(self, name);
 }
 
+PyObject *
+_Py_union_args(PyObject *self)
+{
```

Comment:
Put in an assertion that self is a Union. (It will get elided in non-debug mode.)

Reply:
Fixed, thanks.